From b8172718a125e55c5e4baf91963220a12fce39a4 Mon Sep 17 00:00:00 2001
From: Hongcheng Xie <hongcheng.xie@intel.com>
Date: Tue, 18 Aug 2020 11:18:39 +0800
Subject: [PATCH] Change sdcard bin name as sdcard2

Cic cloud will copy cic private sdcard binary as default
sdcard bin, change original google sdcard bin name to
avoid compile error in R.

Change-Id: Ief3d27d880362e9858b213e91856445ae0d484c7
Tracked-On: ACP-1189
Signed-off-by: Hongcheng Xie <hongcheng.xie@intel.com>
---
 sdcard/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sdcard/Android.bp b/sdcard/Android.bp
index c096587ba..f05f18b44 100644
--- a/sdcard/Android.bp
+++ b/sdcard/Android.bp
@@ -1,6 +1,6 @@
 cc_binary {
     srcs: ["sdcard.cpp"],
-    name: "sdcard",
+    name: "sdcard2",
     cflags: [
         "-Wall",
         "-Wno-unused-parameter",
-- 
2.39.0

